### PR TITLE
Make admin feedback easier understandable

### PR DESCRIPTION
**Pull request by pmaier1**
You might want to change ('**occ upgrade**') to this. Should also be fine if you keep it as it is :)

### CODE REVIEW

File: core/templates/update.use-cli.php [hunk 1]
```diff
@@ -3,9 +3,9 @@
 		<h2 class="title"><?php p($l->t('Update needed')) ?></h2>
 		<div class="infogroup">
 			<?php if ($_['tooBig']) {
-				p($l->t('Please use the command line updater because you have a big instance.'));
+				p($l->t('This ownCloud instance possibly hosts many users and files. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).'));
```
Comment (DeepDiver1975) [inline core/templates/update.use-cli.php]: for white labling reasons owncloud has to be replaced with a variable - or you simply remove it.
 this -> your

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: core/templates/update.use-cli.php (hunk 1 above)
Comment (pmaier1) [inline core/templates/update.use-cli.php]: - 'This' changed to 'your'
- 'ownCloud' removed

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by DeepDiver1975** [comment]
@tomneedham please review the english strings - thx

File: core/templates/update.use-cli.php [hunk 2]
```diff
@@ -3,9 +3,9 @@
 		<h2 class="title"><?php p($l->t('Update needed')) ?></h2>
 		<div class="infogroup">
 			<?php if ($_['tooBig']) {
-				p($l->t('Please use the command line updater because you have a big instance.'));
+				p($l->t('Your instance possibly hosts many users and files. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).'));
 			} else {
-				p($l->t('Please use the command line updater because automatic updating is disabled in the config.php.'));
+				p($l->t('Automatic updating was disabled in config.php. To upgrade your instance, please use the command line updater (occ upgrade).'));
```
Comment (tomneedham) [inline core/templates/update.use-cli.php]: Automatic updating is disabled in config.php. To upgrade your instance, please use the command line updater (occ upgrade)

File: core/templates/update.use-cli.php [hunk 3]
```diff
@@ -3,9 +3,9 @@
 		<h2 class="title"><?php p($l->t('Update needed')) ?></h2>
 		<div class="infogroup">
 			<?php if ($_['tooBig']) {
-				p($l->t('Please use the command line updater because you have a big instance.'));
+				p($l->t('Your instance possibly hosts many users and files. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).'));
```
Comment (tomneedham) [inline core/templates/update.use-cli.php]: It looks like your instance may host many files and/or users. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).

**Review by tomneedham — CHANGES_REQUESTED** [review]

**Comment by pmaier1** [comment]
Thanks for review @tomneedham. Incorporated the requested changes.

**Review by tomneedham — APPROVED** [review]

**Comment by PVince81** [comment]
stable10: https://github.com/owncloud/core/pull/28828

**Pull request merged**